src/consensus/consensus.h

```diff
@@ -27,6 +27,12 @@ static const size_t MIN_SERIALIZABLE_TRANSACTION_WEIGHT = WITNESS_SCALE_FACTOR *
 /** Interpret sequence numbers as relative lock-time constraints. */
 static constexpr unsigned int LOCKTIME_VERIFY_SEQUENCE = (1 << 0);
 
+/**
+ * Defines how many seconds earlier the timestamp of the first block in a difficulty adjustment
+ * period can be compared to the last block of the previous period (BIP54).
+ */
+static constexpr int64_t MAX_TIMEWARP{600 * 12};
```

Comment:
87e9e658b74da7ce0e1134c59563bc88b8820f37

micro-nit: this calculation doesn't make it easier to match the BIP text which just says 7200

Reply:
comment more than sufficient

Reply:
Updated the constant value to `2 * 60 * 60`. Updated the comment to mention 7200 (had to rephrase it a bit).